scripts/mkcloud

```diff
@@ -666,7 +668,7 @@ function onhost_create_libvirt_computenode_config()
     cephvolume="$4"
     drbdvolume="$5"
     nicmodel=virtio
-    hypervisor_has_virtio || nicmodel=e1000
+    hypervisor_has_virtio "$6" || nicmodel=e1000
```

Comment:
What is $6? It should be assigned to a variable with a proper name to make it understandable.